internal/validations/observability/outputs/splunk_test.go

```diff
@@ -0,0 +1,65 @@
+package outputs
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	obs "github.com/openshift/cluster-logging-operator/api/observability/v1"
+)
+
+var _ = Describe("validating Splunk", func() {
+	Context("#ValidateSplunkMetadata", func() {
+
+		It("should fail validation if meet indexed field not in payload", func() {
```

Comment:
If i read this validation correctly, the "event" is always the entire vector log event (e.g. `.`)?  And you can only index fields that are part of what gets added to `.`?  The splunk docs would suggest otherwise but I guess vector's encoding does not allow defining the "event" content separate from "indexed field values"?  Maybe the "indexed" fields at the root level would need to be added to the "exclude" fields?

Reply:
This checks not related to the Splunk requirements directly, more about our procedure how we remap/calculate fields. If indexed fields will be outside the given payload, we can't remap it correctly. So this warning to check given settings. I think adding "indexed" fields to exclude filter not make sens, we have `.log_source` and `.log_type` these should be enough.

Reply:
See my previous comment about the validation